### ISSUE TITLE
Deactivate 

### ISSUE THREAD

**Issue by jordandark** [4w4k3/BeeLogger#6]
how i deactivate this in my infected computer

**Comment by 4w4k3** [comment]
Type: msconfig and disable it.

**Comment by evaristegal0is** [comment]
After I disabled beelogger in msconfig, can I re-enable and re-use it?
And after I disabled beelogger, have I to delete something to not leave trace?

**Comment by 4w4k3** [comment]
Beelogger isn't to black hat, its a tool designed to pentest, so he's not been wrote to clear traces, you need learn how to this with much search and dedication and i really hope that you not are using this for illegal purposes.

**Comment by evaristegal0is** [comment]
No, simply I have not yet had time to read me all the files. I have only tried beelogger because I didn't think it would work without being seen dall'antivirus, instead it works.
Except with Avast, Avast recognizes and blocks it.
I thought that Beelogger was like a metasploit payload, and that after a reboot it stops to work, but it is not.
Thanks for your answer!

**Comment by evaristegal0is** [comment]
So, I wanted just to remove or disable it permanently from my computer, nothing of illegal.

**Comment by 4w4k3** [comment]
First delete the key in regedit and after find in Users\Public\ for  adobe and delete the exe.

**Comment by evaristegal0is** [comment]
Beelogger continue to send me email. I don't find the process in msconfig, and I don't find the key in regedit, exscuse me for embarrassing failure

**Comment by evaristegal0is** [comment]
So, I have deleted the key and the exe but after reboot it is disabled but present in msconfig. I can't remove it totally.

**Comment by 4w4k3** [comment]
If you delete right he can't work, please check out again 
First delete the key in regedit and after find in Users\Public\ for adobe and delete the exe

**Comment by jordandark** [comment]
it´s late to say this but thanks #FORTHELULZ
